packages/scalars/package.json

```diff
@@ -28,10 +28,6 @@
     "graphql": "^16.9.0",
     "zod": "^3.23.8"
   },
-  "peerDependencies": {
```

Comment:
Not sure why these are peerDependencies.
There is a nasty issue that has come up in the past due to having different versions of graphql being loaded: https://stackoverflow.com/questions/70918391/duplicate-graphql-modules-cannot-be-used-at-the-same-time
Just to keep in mind in case this error reappears